.github/workflows/push.yaml

```diff
@@ -3,7 +3,7 @@ name: Build container images
 on: # yamllint disable-line rule:truthy
   push:
     branches: [main]
-  pull_request_target:
```

Comment:
This also needs to be reverted; we need to have the `pull_request_target` trigger for things to work.

Reply:
done